### PR TITLE
Fix gcc-11.x.x compilation warnings considered as fatal

### PR DESCRIPTION
**Pull request by ROBERT-MCDOWELL**
...

### CODE REVIEW

File: src/mod/databases/mod_pgsql/mod_pgsql.c [hunk 1]
```diff
@@ -774,6 +774,11 @@ switch_status_t pgsql_next_result_timed(switch_pgsql_handle_t *handle, switch_pg
 		case PGRES_COPY_IN:
 		case PGRES_COMMAND_OK:
 			break;
+//#if POSTGRESQL_MAJOR_VERSION >= 14
```
Comment (andywolk) [inline src/mod/databases/mod_pgsql/mod_pgsql.c:777]: not going to build on older versions

Comment (ROBERT-MCDOWELL) [reply src/mod/databases/mod_pgsql/mod_pgsql.c:777]: Do you know how to put this condition out of the switch since gcc-11.2.x does not allow it.

Comment (ROBERT-MCDOWELL) [reply src/mod/databases/mod_pgsql/mod_pgsql.c]: ok, I fixed the conflicts with master, but anyhow it cannot compile with gcc-11.x.x for now

Comment (ROBERT-MCDOWELL) [reply src/mod/databases/mod_pgsql/mod_pgsql.c]: ok my bad, it seems that now the issue went away. consider this patch ok.
however the other issue I posted is still persisting (#1440 )

Comment (ROBERT-MCDOWELL) [reply src/mod/databases/mod_pgsql/mod_pgsql.c]: I sent another PR that fixes the problem on the #if #endif POSTGRESQL_MAJOR condition into the switch.
gcc 11.2 does not allow a condition into a switch unless the "default: break;" is specified.
everything is compiling fine now (with all the patches I sent)

**Review by ROBERT-MCDOWELL — COMMENTED** [review]
fix compliation with gcc 11.2.x

**Comment by dragos-oancea** [comment]
we merged  gcc-11 related fixes, but only for the core (b4a7cf40597a7746e10d11e5bc6b3b6ee3fb0af6) . please approach the module fixes in a separate PR if you wish.

**Pull request closed without merging**